services/notification/notifiers/mixins/message/__init__.py

```diff
@@ -1,13 +1,12 @@
 import logging
-from typing import Callable, List
+from typing import Callable
```

Comment:
[nit] `typing.Callable` is deprecated in favor of `collections.abc.Callable` (see [here](https://docs.python.org/3/library/typing.html#annotating-callable-objects))